index.html

```diff
@@ -1654,6 +1925,381 @@ <h2>
+    Additionally, the [=extension command=] can be used to check whether a source type is created
+    via the [[[#create-virtual-pressure-source]]] [=extension command=] by checking the response.
+    If it has not been created the response will contain the [=error=] with
+    [=error code|WebDriver error code=]
+    <a data-cite="webdriver2/#dfn-unsupported-operation">unsupported operation</a>.
+  </aside>
+  <table class="data">
+    <caption>
+      Properties of the parameters argument used by this algorithm
+    </caption>
+    <tr>
+      <th>
+        Parameter name
+      </th>
+      <th>
+        Value type
+      </th>
+      <th>
+        Required
+      </th>
+    </tr>
+    <tr>
+      <td>
+        sample
+      </td>
+      <td>
+        {{PressureState}}
+      </td>
+      <td>
+        yes
+      </td>
+    </tr>
+  </table>
+  <p>
+    The [=remote end steps=] given |session|, |URL variables| and |parameters| are:
+  </p>
+  <ol class="algorithm" data-cite="WEBDRIVER2">
+    <li>
+      Let |virtualPressureSourceType| be the value of the |URL variables|["type"].
+    </li>
+    <li>
+      If |virtualPressureSourceType| is not a [=valid source type=], return [=error=] with [=error code|WebDriver error code=] [=invalid argument=].
+    </li>
+    <li>
+      Let |topLevelTraversable| be the <a data-cite="webdriver2/#dfn-current-browsing-context">current browsing
+      context</a>'s [=browsing context/top-level traversable=].
+    </li>
+    <li>
+      Let |topLevelVirtualPressureSourceMapping| be the |topLevelTraversable|'s [=virtual pressure source mapping=].
+    </li>
+    <li>
+      If |topLevelVirtualPressureSourceMapping| does not [=map/contain=] |virtualPressureSource|,
+      return [=error=] with [=error code|WebDriver error code=]
+      <a data-cite="webdriver2/#dfn-unsupported-operation">unsupported operation</a>.
+    </li>
+    <li>
+      Let |virtualPressureSource| be |topLevelVirtualPressureSourceMapping|[|virtualPressureSourceType|].
+    </li>
+    <li>
+      Let |sample| be the result of invoking
+      <a data-cite="!WEBDRIVER2#dfn-getting-properties">get a property</a> "sample" from |parameters|.
+    </li>
+    <li>
+      If |sample| is not of type {{PressureState}}, return [=error=] with [=error code|WebDriver error code=] [=invalid argument=].
+    </li>
+    <li>
+      Set |virtualPressureSource|'s [=virtual pressure source/pending sample=]
+      to a new [=pending sample descriptor=] whose [=pending sample
+      descriptor/state=] is |sample| and [=pending sample
+      descriptor/timestamp=] is the [=unsafe shared current time=].
+    </li>
+    <li>
+      In an [=implementation-defined=] way, make |virtualPressureSource|'s
+      [=virtual pressure source/pending sample=] available to
+      |virtualPressureSource|'s [=virtual pressure source/connected platform
+      collectors=].
+    </li>
+    <li>
+      Return [=success=] with data <code>null</code>.
```

Comment:
null is not within `<code>` in other places

Reply:
Fixed in commit e63d26b.